layouts/community/ortho_4x10/scotto/keymap.c

```diff
@@ -0,0 +1,75 @@
+/*
+Copyright 2023 Joe Scotto
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation, either version 2 of the License, or
+(at your option) any later version.
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+GNU General Public License for more details.
+You should have received a copy of the GNU General Public License
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+#include QMK_KEYBOARD_H
+#include "scotto.h"
```

Comment:
Since "scotto.h" has the qmk keyboard h included, you don't need to include it here.
```suggestion
#include "scotto.h"
```